kola/tests/update/update.go

```diff
@@ -211,3 +222,86 @@ func splitNewlineEnv(envs string) map[string]string {
 	}
 	return m
 }
+
+func sysextBootLogic(c cluster.TestCluster) {
```

Comment:
Could you split the `noIgnition` and `withIgnition` into separate subtests? I think you could do it like:
```go
c.Run("no ignition", func(c cluster.TestCluster) { … }))
c.Run("with ignition", func(c cluster.TestCluster) { … }))
```

Reply:
Not sure, the `version` variable resolved in one is used in the second

Reply:
Ok, let's keep it as is then.